webapp/src/components/DeleteUserPage/DeleteUserPage.vue

```diff
@@ -0,0 +1,398 @@
+              <img v-else :src="icon" alt="Icon" class="avatar-icon" />
+            </div>
+            <span class="member-name">{{ member.name }}</span>
+          </div>
+          <button 
+            @click="deleteMember(member)" 
+            class="delete-button"
+            :title="'Delete ' + member.name"
+          >
+            ×
+          </button>
+        </div>
+        <!-- Show More Button -->
+        <button 
+          v-if="hasMoreMembers" 
+          @click="showMore" 
+          class="show-more-button"
+        >
+          Show More...
+        </button>
+      </div>
+    </div>
+
+    <!-- Delete User Button -->
+    <div class="delete-user-container">
+      <div class="delete-user-content">
+        <button 
+          @click="deleteSelectedUser" 
+          class="delete-user-button"
+          :disabled="!selectedMember"
+        >
+          Delete User
+        </button>
+        <!-- Group Icon image -->
+        <img src="@/assets/img/groups-ico.png" alt="groups icon" class="groups-icon-image" />
+      </div>
+    </div>
+  </div>
+</div>
+</div>
+</template>
+
+<script>
+import icon from '@/assets/img/user-blue-ico.png';
+import NavBar from "../NavBar.vue";
+//import router from '@/router';
+
+export default {
+name: 'Group',
+components: {
+NavBar
+},
+data() {
+  return {
+    members: [
+      { id: 1, name: 'User 1', avatar: icon },
+      { id: 2, name: 'User 2', avatar: icon },
+      { id: 3, name: 'User 3', avatar: icon },
+      { id: 4, name: 'User 4', avatar: icon },
+      { id: 5, name: 'User 5', avatar: icon },
+      { id: 6, name: 'User 6', avatar: icon },
+      { id: 7, name: 'User 7', avatar: icon },
+      { id: 8, name: 'User 8', avatar: icon },
+    ],
+    searchQuery: '',
+    displayLimit: 3,
+    selectedMember: null,
+    searchTimeout: null,
+  };
+},
+
+computed: {
+  filteredMembers() {
+    if (!this.searchQuery) {
+      return this.members;
+    }
+    
+    const query = this.searchQuery.toLowerCase().trim();
+    return this.members.filter(member => 
+      member.name.toLowerCase().includes(query)
+    );
+  },
+  
+  displayedMembers() {
+    return this.filteredMembers.slice(0, this.displayLimit);
+  },
+
+  hasMoreMembers() {
+    return this.filteredMembers.length > this.displayLimit;
+  }
+},
+
+methods: {
+  handleSearch(event) {
+    // remoces previous timeout setting
+    if (this.searchTimeout) {
+      clearTimeout(this.searchTimeout);
+    }
+
+    // new timeout
+    this.searchTimeout = setTimeout(() => {
+      this.searchQuery = event.target.value;
+      // resets display limit after update
+      this.displayLimit = 3;
+    }, 300);
+  },
+
+  showMore() {
+    this.displayLimit += 3;
+  },
+
+  deleteMember(member) {
+    this.selectedMember = member;
+  },
+
+  deleteSelectedUser() {
+    if (this.selectedMember) {
+      fetch(import.meta.env.VITE_API_URL + "/deleteUser", {
```

Comment:
/deleteUser is not a route

Reply:
fixed routing and added fetch method

Reply:
will adjust the fetch method accordingly after other pages are done. Currently it just gives the 400 error since users are not added yet.